src/components/homePage/Speakers.vue

```diff
@@ -1,19 +1,136 @@
 <template>
 <div id="speakers">
-	<h2>CUTC 2020</h2>
-	<!-- TAB LAYOUT -->
+	<div class="tabs">
+		<p class="tab" :class="[currentPage == 0 ? 'underlined' : '']" @click="currentPage = 0; toggleStyle()"> Speakers </p>
+		<p class="tab" :class="[currentPage == 1 ? 'underlined' : '']" @click="currentPage = 1; toggleStyle()"> Events </p>
+	</div>
+  
+	<vueper-slides v-if="currentPage == 0">
+		<template v-slot:arrow-right>
+			<img src="../../assets/Arrow.svg"/>
+		</template>
+		<vueper-slide 
+			v-for="(slide, i) in information['slides']" 
+			:key="i" >
+			<template v-slot:content>
+                  <img :src="getImage(slide.image)" id="person" :alt="slide.alt"/>
+                {{ slide.content }}
+        <div class="vueperslide__content-wrapper">
+        </div>
+      </template>
+		</vueper-slide>
+	</vueper-slides>
+	<vueper-slides v-else>
+		<template v-slot:arrow-right>
+			<img src="../../assets/Arrow.svg"/>
+		</template>
+		<vueper-slide 
+			v-for="(slide, i) in information['slides']" 
+			:key="i" >
+			<template v-slot:content>
+                  <img :src="getImage(slide.image)" id="person" :alt="slide.alt"/>
+                {{ slide.content }}
+        <div class="vueperslide__content-wrapper">
+        </div>
+      </template>
+		</vueper-slide>
+	</vueper-slides>
```

Comment:
We forgot to ask design what the event section should look like (that's my bad), so it's fine that you copy and pasted this section. However, the `veuper-slide` for-loop uses the **exact** same information as the speaker section, meaning it's content is going to be identical. To fix this you'll need to create two different sections in the `speakers.json` file like this:

`
{
  "speakers" : [...],
  "events" : [...],
}`

Let me know if this makes sense and if you need and help making the changes!